src/main/java/org/ibp/api/rest/study/StudyResource.java

```diff
@@ -70,4 +75,55 @@
 		final List<TreeNode> studyTree = this.studyService.getStudyTree(parentFolderId, programUUID);
 		return new ResponseEntity<>(studyTree, HttpStatus.OK);
 	}
+
+	@ApiOperation(value = "Get study entries",
+		notes = "Get study entries as table")
+	@ApiImplicitParams({
+		@ApiImplicitParam(name = "page", dataType = "integer", paramType = "query",
+			value = "Results page you want to retrieve (0..N)"),
+		@ApiImplicitParam(name = "size", dataType = "integer", paramType = "query",
+			value = "Number of records per page."),
+		@ApiImplicitParam(name = "sort", allowMultiple = true, dataType = "string", paramType = "query",
+			value = "Sorting criteria in the format: property(,asc|desc). " +
+				"Default sort order is ascending. " +
+				"Multiple sort criteria are supported.")
+	})
+	@RequestMapping(value = "/{cropName}/programs/{programUUID}/studies/{studyId}/entries", method = RequestMethod.GET)
```

Comment:
Why not include it in `StudyGermplasmResouce` (to be renamed to StudyEntryResource) to consolidate germplasm/entries services?